aws/resource_aws_elasticsearch_domain_test.go

```diff
@@ -349,6 +395,52 @@ resource "aws_elasticsearch_domain" "example" {
 `, randInt, instanceInt, snapshotInt)
 }
 
+func testAccESDomainConfig_WithVPCOptions(randInt int) string {
+	return fmt.Sprintf(`
+resource "aws_vpc" "foo" {
+	cidr_block = "10.10.0.0/16"
+}
+
+resource "aws_subnet" "foo" {
+	cidr_block = "10.10.1.0/24"
+	vpc_id = "${aws_vpc.foo.id}"
+}
+
+resource "aws_security_group" "foo" {
+  name = "terraform_acceptance_test_example"
```

Comment:
Can you add randomization to this name also? it will help avoiding conflicts when running acceptance tests on our side 😄